.ci/ci.R

```diff
@@ -5,10 +5,10 @@
 ## added ver argument to produce R version independent urls
 ## https://bugs.r-project.org/bugzilla3/show_bug.cgi?id=17420
 contrib.url <-
-function (repos, type = getOption("pkgType"), ver) 
+function (repos, type = getOption("pkgType"), ver)
```

Comment:
Could we revert those changes? It doesn't really fit this PR and it is polluting history of those files.

Reply:
am i the only one with the pre-commit hook blocking trailing whitespace / multiple trailing newlines?

i added these since i was being blocked from committing.  maybe it's something i set up on this machine only...